src/components/calculator.css

```diff
@@ -0,0 +1,107 @@
+.calculator {
+  display: grid;
+  grid-template-rows: auto auto repeat(5, 1fr);
+  grid-template-columns: repeat(4, 1fr);
+  width: 500px;
+  height: 400px;
+  grid-gap: 5px;
+  padding: 10px;
+  border: 1px solid black;
+}
+
```

Comment:
- [ ] As per [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) you are not supposed to add any extra design. But when I compare your design with the given design, I see differences in
- the gaps between the buttons and the border
- the orange color you have used for the operation buttons
- the display of the answer has two zeroes, one up and one down
Check the attached screenshots below and make your design to match the given design
- Given design
![image](https://user-images.githubusercontent.com/62849872/227150931-c19f8def-f7cc-4ed6-8ddf-35299775e654.png)

- Your design
![image](https://user-images.githubusercontent.com/62849872/227151178-d81b1db6-37a2-4ec6-bfb8-9fae12d0eb2c.png)

Reply:
@t-yanick Hey can you help me out with the color code for the orange used? I have no idea how to determine what specific color was used

Reply:
- [ ] You can use this color [picker](https://imagecolorpicker.com/en) and do a screenshot of a button and upload on the site to get the exact color. Hope this helps.